CSS/styles.css

```diff
@@ -1,71 +1,79 @@
 :root {
   --almost-black: #252121;
+  --GhostWhite: #F8F8FF;
 }
 
-div{
+.aboutMe {
+  float: left;
+  display: inline-block;
+}
+
+.bottomContainer {
+  background-color: yellow;
+}
+
+.container-h1 {
+  font-size: 2rem;
+  top: 150px;
+  right: 110px;
+}
+
+.container-h2 {
+  font-size: 1.5rem;
+}
+
+.container-p {
+  font-size: 1rem;
+  top: 210px;
+  right: 30px;
+}
+

```

Comment:
los van of de classes `container-h1`, `container-h2` etc nodig zijn: de benamingen kloppen niet helemaal.

je zet de class `container-h1` namelijk op een `<h1>`. maar container impliceert dat *binnen* de container een h1 valt. bijvoorbeeld:

```
<div class="h1-container">
   <h1>blabla</h1>
</div>
```